README.md

```diff
@@ -828,12 +828,27 @@ print(string.isNumber) // false
 Check if it's a valid email:
 
 ```swift
+// (deprecated)
 var string = "test@gmail.com"
 print(string.isEmail) // true
 var string = "test@"
 print(string.isEmail) // false
 ```
```

Comment:
No need to keep it referenced since it's deprecated 

```suggestion
```

Reply:
I would suggest keeping it around until you bump the minimum os versions to the point that you can actually drop the code. Otherwise it might be unclear that the utility even exists for devs who have to support older os versions.